resources/lang/fr/disposable.php

```diff
@@ -3,78 +3,75 @@
     '100ll_cost'     => 'Coût 100LL',
-    'avg_taxi_times' => 'Moyenne du Temps au Taxi',
-    'gh_cost'        => 'Coût de la Manutention au Sol',
+    'avg_taxi_times' => 'Moyenne de durée Taxi',
+    'gh_cost'        => 'Coût du service au Sol',
     'icao_iata_code' => 'ICAO / IATA Code',
     'jeta1_cost'     => 'Coût JET-A1',
     'mogas_cost'     => 'Coût Mogas',
     // Roster
     'awards'      => 'Récompenses',
-    'only_active' => 'Seuls les membres actifs sont répertoriés. La limite inactif est de :days jours',
+    'only_active' => 'Seuls les membres actifs sont répertoriés. Vous serez radié au bout de :days jours d\'absence.',
     'rank'        => 'Grade',
     // Pireps
-    'fused'  => 'Carburant Utilisé',
-    'lpirep' => 'Dernier détails PIREP',
-    'lrate'  => 'Landing Rate',
+    'fused'  => 'Consommé',
+    'lpirep' => 'Dernier PIREP soumis:',
+    'lrate'  => 'Tx de descente',
     'score'  => 'Score',
     // Dashboard
-    'avg_ftime'     => 'Moy.Temps de Vol',
-    'avg_fused'     => 'Moy.Carburant Utilisé',
-    'avg_lrate'     => 'Moy.Landing Rate',
-    'avg_score'     => 'Moy.Score',
-    'user_on_leave' => 'Vous êtes en congé ! Déposez un PIREP pour que votre statut devienne actif.!',
+    'avg_ftime'     => 'Durée moy. de vol',
+    'avg_fused'     => 'Consommation moyenne',
+    'avg_lrate'     => 'Descente: tx moy.',
+    'avg_score'     => 'Score Moyen',
+    'user_on_leave' => 'Vous êtes de repos ! Déposez un PIREP pour vous déclarer actif!',
     // Menu Items
-    'menu_roster'   => 'Liste des Pilotes',
+    'menu_roster'   => 'Nos Pilotes',
     'menu_airlines' => 'Compagnies',
     'menu_ranks'    => 'Grades',
     'menu_awards'   => 'Récompenses',
     'menu_stats'    => 'Statistiques',
-    'menu_fltops'   => 'Operations de vols',
+    'menu_fltops'   => 'Opérations de vols',
     'menu_flights'  => 'Vols',
-    'menu_tours'    => 'Tours',
-    'menu_fleet'    => 'Flotte',
-    'menu_hubs'     => 'Bases',
-    'menu_reports'  => 'Rapports Pilotes',
+    'menu_tours'    => 'Circuits',
+    'menu_fleet'    => 'Flottes',
+    'menu_hubs'     => 'Aéroports Hubs',
+    'menu_reports'  => 'Débriefs Pilotes',
     'menu_mapflt'   => 'Vols en cours',
     'menu_mapwx'    => 'Météo réelle',
-    'menu_docs'     => 'Docs & Resources',
+    'menu_docs'     => 'Docs & Ressources',
     'menu_notams'   => 'NOTAMs',
     'menu_news'     => 'Nouvelles',
-    'menu_opsman'   => 'Manuel Opérationnel',
+    'menu_opsman'   => 'Mode opératoire',
     'menu_profile'  => 'Profil',
     'menu_company'  => 'Compagnie',
-    'menu_base'     => 'Base',
-    'menu_assign'   => 'Assignements',
+    'menu_base'     => 'Basé',
+    'menu_assign'   => 'Votre Planning',
     'menu_myflight' => 'Vol Libre',
-    'menu_bids'     => 'Enchères',
-    'menu_pireps'   => 'Rapports',
-    'menu_market'   => 'Boutique Pilote',
-    'menu_mymarket' => 'Articles Achetes',
+    'menu_bids'     => 'Projets',
+    'menu_pireps'   => 'Débriefs',
```

Comment:
I think we can use "Rapports" (reports) or "PIREP" to talk about a PIREP but "Débrief" (debrief) doesn't means anything to me